guides/doc-Provisioning_Guide/topics/Cloud-Amazon_EC2.adoc

```diff
@@ -247,10 +271,20 @@ EOS
 . Click the *Templates* tab, and from the *Finish Template* list, select your finish template.
 . Navigate to *Hosts* > *Create Host* and enter the information about the host that you want to create.
 . Click the *Parameters* tab and navigate to *Host parameters*.
+ifeval::["{Build}" == "satellite"]
 . In *Host parameters*, click the *Add Parameter* button three times to add three new parameter fields. Add the following three parameters:
+endif::[]
+ifeval::["{Build}" == "foreman"]
+. If you have the Remote Execution plugin installed, in *Host parameters*, click the *Add Parameter* button two times to add two new parameter fields. If you use the Katello plugin, add a third parameter field. Add the following parameters:
+endif::[]
```

Comment:
I think this is sane way of doing things. Does not look so bad in the adoc source. Good.